.github/workflows/extra_tests.yml

```diff
@@ -53,6 +53,8 @@ jobs:
           # See https://github.com/shadow/shadow/issues/2166
           ref: ${{ github.event.pull_request.head.sha }}
           persist-credentials: false
+      - name: Set Rust toolchain
+        run: ln -s shadow/ci/rust-toolchain-stable.toml rust-toolchain.toml
```

Comment:
Are you sure we don't want the `rust-toolchain.toml` to be sitting in the CWD where we initiate the shadow build, which I think is the shadow base dir? (We `cd` into `shadow/` before running the build script.)

Reply:
Yes, the ancestor directories are also searched for `rust-toolchain.toml`, so this affects the `shadow` directory as well.